src/lib/outgoing_message.ts

```diff
@@ -16,6 +16,7 @@ export type JSONValue =
   | number
   | boolean
   | null
+  | undefined
```

Comment:
JSON does not have `undefined`. In fact, it's the greatest hack in JavaScript history, as undefined keys are removed from the output:

```
> JSON.stringify({hello: undefined})
"{}" 
```

Reply:
yeah it was a typing issue with the result. I could use `as any` instead